api/src/main/java/org/apache/cloudstack/api/command/admin/iso/ListIsoPermissionsCmdByAdmin.java

```diff
@@ -1,4 +1,4 @@
-// Licensedname = "listIsoPermissions",  to the Apache Software Foundation (ASF) under one
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@DaanHoogland Just curious why the rat check didn't pick this up. Do we need any update in the github action?

Reply:
maybe, it's not checking the license content

Reply:
Or it found the needed string and the extra bit doesn’t matter…?